src/components/map-line/map-line.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Prop
+} from "@stencil/core";
+import { Component as GxComponent } from "../common/interfaces";
+import { polyline } from "leaflet/dist/leaflet-src.esm";
+import { parseCoords } from "../common/coordsValidate";
+
+@Component({
+  shadow: false,
+  //styleUrl: "map-line.scss",
+  tag: "gx-map-line"
+})
+export class MapLine implements GxComponent {
+  @Element() element: HTMLGxMapLineElement;
+  private lineInstance: any;
+
+  /**
+   * The coordinates where the polyline will appear in the map.
+   *
+   */
+  @Prop({ mutable: true }) coords = "0, 0";
+
+  /**
+   * Emmits when the element is added to a `<gx-map>`.
+   *
+   */
+  @Event() gxMapLineDidLoad: EventEmitter;
+
+  /**
+   * Emmits when the element is deleted from a `<gx-map>`.
+   *
+   */
+  @Event() gxMapLineDeleted: EventEmitter;
+
+  private setupLine(coords) {
+    this.lineInstance = polyline(coords, {
+      color: "red",
+      weight: 3
```

Comment:
I feel like these two should be properties of the component, to allow customization, but we can improve this in a future PR.

Reply:
I agree.